rock-paper-scissors.js

```diff
@@ -0,0 +1,78 @@
+function computerPlay() {
+  let computerHand = Math.floor(Math.random() * 3);
+  if (computerHand === 0) {
+    return "rock";
+
+  }
+  else if (computerHand === 1) {
+    return "paper";
+
+  }
+  else if (computerHand === 2) {
+    return "scissors";
+
+  }
+}
+
+function playRound(playerSelection, computerSelection) {
+  playerSelection = playerSelection.toLowerCase();
+  if (playerSelection === "rock") {
```

Comment:
A little way this could be improved is to check if the computer and player selected the same before going through the if statements and return the draw text.